MAUIsland/Features/Gallery/MAUI/Services/Implementations/MAUIControlsService.cs

```diff
@@ -38,6 +38,12 @@ public Task<IEnumerable<ControlInfo>> GetAllControlInfoAsync()
                 ControlRoute = AppRoutes.ProgressBarPage
             });
 
+            controls.Add(new ControlInfo()
+            {
+                ControlName = "Slider",
+                ControlRoute = AppRoutes.SliderPage
```

Comment:
Write like this 
```
            controls.Add(new ControlInfo()
            {
                ControlName = "Date Picker",
                ControlRoute = AppRoutes.DatePickerPage,
                ControlIcon = new FontImageSource()
                {
                    FontFamily = FontNames.FluentSystemIconsRegular,
                    Glyph = FluentUIIcon.Ic_fluent_calendar_3_day_28_regular
                },
                ControlDetail = "DatePicker invokes the platform's date-picker control and allows you to select a date."
            });
```

Reply:
Slider is a horizontal bar that you can manipulate to select a double value from a continuous range.